modules/external/index.ts

```diff
@@ -12,17 +23,14 @@ app.use(errorHandler());
 /**
  * Start Express server.
  */
-const server = app.listen(app.get("port"), () => {
+const server = app.listen(port, () => {
   console.log(
     '  App is running at http://localhost:%d in %s mode',
-    app.get("port"),
+    port,
     app.get("env")
   );
   console.log('  Press CTRL-C to stop\n');
 });
 
-
-
 bot.setWebsocket(socket(server));
 
-export default server;
```

Comment:
I renamed this file to index.ts as I think it makes it easier to understand the flow. It's the entrypoint and common practise for it to be the index